sdcm/cluster.py

```diff
@@ -2566,6 +2566,83 @@ def kill_stress_thread_bench(self):
                 if kill_result.exit_status != 0:
                     self.log.error('Terminate scylla-bench on node %s:\n%s', loader, kill_result)
 
+    def run_fullscan(self, loader_node, db_node):
+        """Run cql select count(*) request
+
+        Get all non-system keyspaces
+        Get all tables from randomly choosen ks
+        Request selecl count(*) for ks and randomly choosen cf
+
+        Arguments:
+            loader_node {BaseNode} -- loader cluster node
+            db_node {BaseNode} -- db cluster node
+
+        Returns:
+            object -- object with result of remoter.run command
+        """
+        cmd = 'describe keyspaces'
+        result = loader_node.remoter.run('cqlsh --execute "{}" {}'.format(cmd, db_node.private_ip_address), verbose=False)
```

Comment:
using `private_ip_address` can be a problem when running in multi-datacenter mode

Reply:
you should use `node.ip_address(...)`

Reply:
See example in `configure_scylla_monitoring`

Reply:
Fixed.